### PR TITLE
Added new section to tech stacks: introduction to framer motion.

### PR DESCRIPTION
**Pull request by CharlieMorocz**
See issue #454

Added a new section, "Intro to Framer Motion, an Animation Library for React" to the tech stacks subcategory.

* Provided an introduction and overview of what Framer Motion is and how to install it
* Covered the fundamental concepts and components of Framer Motion, included reference code blocks and associated gifs
* Provided links to helpful resources

### CODE REVIEW

File: Topics/Tech_Stacks/Framer_Motion.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+- #### [Keyframes](#keyframes)
+- #### [Variants](#variants)
+### [Conclusion](#conclusion)
+### [Resources](#resources)
+
+
+## Introduction & Prerequisites
+Framer Motion is a production-ready animation library designed for React. Framer Motion simplifies the process of adding animations by using unique components and props which act as an interface for adding animated properties to elements. 
+
+This page will serve as quick, easy, and informative guide to getting started with the Framer Motion animation library. It will cover the first time installation and set-up of Framer Motion, foundational knowledge of animation components & props, an overview of a few key modifiable values, and a couple of quick tricks. A list of resources will also be included at the end.
```
Comment (selintasman) [inline Topics/Tech_Stacks/Framer_Motion.md:19]: You forgot the "a" on line 19. The sentence should be "This page will serve as a quick, easy, and informative guide..."

File: Topics/Tech_Stacks/Framer_Motion.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+      initial={{ x: "-300px"}}
+      animate={{ x: "300px" }}
+      transition={{ ease: "easeIn", duration: 4 }}
+    />
+```
+
+![Alt Text](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExaWE1NzAzanZxNG0yNHVmZXdiMnJlaHlkczlwaTBzaHZucGV6djZnMCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/gCQUqgks3CufcJIC2k/giphy.gif)
+
+### Keyframes
+Keyframes provide a way to define specific states of an animation at different points in time. By specifying keyframes, you are able to exert more control over the fine details of an animation sequence or create more complex animations. To specify keyframes, simply pass in an array of your desired values as an argument to your animation prop. Note that every value in a keyframe array must be of the same type (i.e., number, px, vw, etc.):
```
Comment (selintasman) [inline Topics/Tech_Stacks/Framer_Motion.md:79]: I really like that you are specifying the types that the values inside the keyframe arrays can take.

**Review by selintasman — CHANGES_REQUESTED** [review]
Great job Charlie! As someone who has never worked with Framer Motion, this guide is very intuitive and beginner-friendly! I really liked how you gave GIF examples after each code snippet. These were very helpful in visualizing what the code did. Additionally, I think you covered all the key props like initial, animate, transition, keyframes, and variants. I have only found one tiny mistake towards the top of the file but other than that the wording was very clear. My only suggestion is maybe you can give the names of a few predefined transitions on top of the "ease-in" transition that you give an example of. But overall, fantastic job!

File: Topics/Tech_Stacks/Framer_Motion.md (hunk 2 above)
Comment (CharlieMorocz) [reply Topics/Tech_Stacks/Framer_Motion.md:79]: Thank you! I thought that would help with visualizing the whole process.

File: Topics/Tech_Stacks/Framer_Motion.md (hunk 1 above)
Comment (CharlieMorocz) [reply Topics/Tech_Stacks/Framer_Motion.md]: [6b79590](https://github.com/learning-software-engineering/learning-software-engineering.github.io/pull/515/commits/6b79590bb3a70e6ab569d02509f473cfc1fc4162)

**Review by selintasman — APPROVED** [review]
Thank you for making the changes Charlie! This looks good to me :)

**Pull request merged**